internal/provider/resources/install/install_common.go

```diff
@@ -0,0 +1,174 @@
+package installresources
```

Comment:
not sure what the style is here; all the TF-provided packages are munged together (e.g. stringvalidators, objectvalidators, etc)

Reply:
I just noticed that, since we're in terraform I suppose it makes more sense to use the munged together forms. but now my spell checker is freaking out. 🤔